File: Data Structures & Algorithms/pow-x-n/submission-1.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        res = 1
        if n>0:
            while(n):
                res = res*x
                n-=1
        elif n<0:
            while(n):
                res = res*(1/x)
                n+=1
        return res

File: Data Structures & Algorithms/maximum-subarray/submission-1.java
class Solution {
    public int maxSubArray(int[] nums) {
        
        int res=nums[0];
        int l = nums.length;
        int currsum = nums[0];

        for(int i =1; i<l; i++){
            currsum = Math.max(currsum + nums[i], nums[i]);
            res = Math.max(currsum,res);
            
        }

        return res;

    }
}
